docs/UserGuide.md

```diff
@@ -406,30 +406,28 @@ Command Words Accepted: `edit_transaction`, `edit`, `et` (case-insensitive)
-| dt/         | Yes       | Date and time where transaction was made.<br/>Defaults to current date time if not specified. | Can be in the `dd-MM-uuuu HH:mm` (e.g. 31-10-2023 12:00), `uuuu-MM-dd HH:mm` (e.g. 2023-10-31 22:59), or `dd MMM uuuu HH:mm` (e.g. 31 Oct 2023 22:59) formats. |
-| l/          | Yes       | Location where transaction was made.<br/>Defaults to `'-'` if not specified.                  | There is a 500-character limit                                                                                                                                 |
-| c/          | Yes       | Category tagged to that transaction.<br/>No categories tagged if not specified.               | There is a 15-character limit.<br/>`Category` entered will be converted and stored as all lowercase.<br/>Each transaction can have at most 5 categories.       |
-
-Important notes:
-
-1. While all options besides `INDEX` are optional, **you must specify at least one field** you wish to edit (
-   i.e. `Name`, `Type`, `Amount`, `Datetime`, `Location`, or `Category`).
-2. You must provide a non-empty value for the `Name`, `Type`, and `Amount` fields if you wish to edit them; they cannot
-   be left empty.
-3. The `INDEX` option must be specified first. The order in which you specify the other options does not matter.
-4. Modifying a transaction's category will result in the replacement of all existing categories. 
-   - For example, in the case of a `Transaction` with two existing categories (Entertainment and Hobbies), editing it with `edit INDEX c/Education` will replace all existing categories, leaving the transaction with a single category, Education."
+| Option Name | Optional? | Purpose                                                                                             |
+|-------------|-----------|-----------------------------------------------------------------------------------------------------|
+| INDEX       | No        | (One-based) Index of the transaction to edit.                                                       |
+| n/          | Yes       | Name of the transaction.                                                                            |
+| type/       | Yes       | Transaction type of transaction.<br/>Valid types are `income` and `expense`.                        |
+| amt/        | Yes       | Monetary amount of transaction. Has to be a positive value.                                         |
+| dt/         | Yes       | Date and time where transaction was made.<br/>Defaults to current date time if field is left empty. |
+| l/          | Yes       | Location where transaction was made.<br/>Defaults to `'-'` if field is left empty.                  |
+| c/          | Yes       | Category tagged to that transaction.<br/>Defaults to no categories if field is left empty.          |
```

Comment:
Think maybe it's better to standardize this by removing this section and adding a callout to reference the `#prefix-types` section on top like we have done for add transaction and get total expenditure.

So the "important notes" only contains the key differences between what we've talked about in the prefix constraints section and how the parameters are actually applied, i.e. things like `c/` working to clear the categories